tsconfig.json

```diff
@@ -5,9 +5,9 @@
 		// "noEmit": true, // Useful when running `tsc` directly
 
 		// TODO: Get all these removed
-		"allowJs": true,
+		"allowJs": false,
 		"declaration": false, // Needed because of `allowJs`
-		"strict": false,
+		"strict": true,
```

Comment:
You can just remove it. It's the default for our config.

Reply:
Done.